esx_service/vmdk_ops_test.py

```diff
@@ -722,15 +722,21 @@ def create_default_tenant_and_privileges(self):
 
         if not tenant_uuid:
             logging.debug("create_default_tenant_and_privileges: create DEFAULT tenant")
-            error_info, tenant = auth_api._tenant_create(
+            error_info, auth_mgr = auth_api.get_auth_mgr_object()
+            if error_info:
+                err = error_code.TENANT_CREATE_FAILED.format(auth.DEFAULT_TENANT, error_info.msg)
+                logging.warning(err)
```

Comment:
sorry not following much here .. don't we need to fail the test at this point  if error_info is set?